library/common/glfw/embedder.cc

```diff
@@ -385,6 +402,26 @@ FlutterWindowRef FlutterEmbedderCreateWindow(
   return state;
 }
 
+bool FlutterEmbedderEngineShutdown(FlutterEngineRef engine_ref) {
```

Comment:
FlutterEmbedderShutDownEngine

(Functions should be verbs when possible, and it also makes it parallel ...RunEngine)

Reply:
Yeah I was sort of mimicking FlutterEngineShutdown, but I agree ShutDownEngine is better.